Security/Guidelines/Rationales.mediawiki

```diff
@@ -0,0 +1,46 @@
+The goal of this document is to detail the rationales behind why various technologies and processes are encouraged or discouraged.
+
+<table><tr>
+<td><div style="float:left;" class="toclimit-3">__TOC__</div></td>
+<td valign="top">
+{| class="wikitable"
+|-
+! Document Status !! Major Versions
+|- 
+|  <span style="color:orange;">'''DRAFT'''</span> ||
+* Version 1: gene: creation
+|}
+[[File:OpSec.png|right|300px]]
+</td>
+</tr></table>
+
+= Rationales =
+{| class="wikitable"
+!Topic
+!Rationale
+|-
+|<div id="shared-passwords">[[#shared-passwords|§]] Shared passwords</div>
+|Shared passwords are passwords that more than one person knows or has access to. They're discouraged beacause
+* Use of them makes auditing access difficult because
+** multiple users appear in audit logs as one user and different users actions are difficult to differentiate
+** OpSec must look through all logs that all users who use the shared password caused entries in
+** correlation of events across different systems is impossible if multiple people are creating event records with a single shared account across multiple systems
```

Comment:
"at the same time" since we can sometimes differenciate if we know who used the login at a certain time, and if no one else has logged in